src/app/navbar/navbar.component.ts

```diff
@@ -47,4 +62,29 @@ export class NavbarComponent implements OnInit {
       this.isCollapsed = true;
     }
 
-}
+    getTemperature(): void { 
+      var weatherQuery = `${baseWeatherUrl}?units=${defaultTemperatureUnits}&zip=${zipcode},us&APPID=${API_KEY}`;
+
+      this.http.get(weatherQuery).subscribe(weatherData => {
+        //set default temperature to Fahrenheit
+        this.temperature = String(Math.round((weatherData['main']['temp']))) + "°F";
+        this.weatherIcon = weatherData['weather'][0]['icon'];
+
+        this.fahrenheit = Math.round(weatherData['main']['temp']);
```

Comment:
Just to make this a bit more efficient just set:

let temp = weatherData...
this.fahrenheit = Math.round(temp);
this.celsius =  Math.round(temp - 32efwefwew);

this.temperature = String(this.fahrenheit) + *F
this.weatherIcon ...

Reply:
actually instead of making fahrenheit/celsius ints, make them strings with *F and *C. so you dont even need to concatenate in future